<ui version="4.0" >
 <class>UIManageParsingFile</class>
 <widget class="QDialog" name="UIManageParsingFile" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Manage Parsing File</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="5" >
    <widget class="QTreeView" name="treeView" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Find pattern :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="lineEdit" >
     <property name="minimumSize" >
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="bNew" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>New</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="bDelete" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" >
    <widget class="QPushButton" name="bSave" >
     <property name="text" >
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5" >
    <widget class="QTextEdit" name="textEdit" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
